UM/Qt/Bindings/FileProviderModel.py

```diff
@@ -46,7 +46,8 @@ def update(self) -> None:
             "name"         : "LocalFileProvider",
             "displayText"  : "From Disk",
             "fileProvider" : None,  # it's not loaded via a plugin, so its FileProvider is empty
-            "shortcut"     : "Ctrl+O"
+            "shortcut"     : "Ctrl+O",
+            "priority"     : 99,  # Assign a high value to make sure it appears on top in the File->Open File(s) submenu
```

Comment:
This default makes no sense to me. All file providers get this priority unless they override it, right? Why would all of them need to appear on top?

![Nobody is](https://i.imgflip.com/4w8ouu.jpg)

Reply:
FileProviders get a priority of 0 by default https://github.com/Ultimaker/Uranium/blob/9b3082238711ace62496cfc8e2669395005dcb46/UM/FileProvider.py#L40. That's why the 99 is given only to the LocalFileProvider item of the FileProviderModel.